acceptance/Admin.Specs/Features/Admin.feature

```diff
@@ -0,0 +1,7 @@
+@Admin
```

Comment:
Do we need a negative test here about Not Signed in? 

Given I have opened the Admin Site 
Then I am forced to log in

I would say that it is more important that unauthorised people cannot access the Admin site before ... I know that this is the first test case but wanted to make sure that we don't miss this

Reply:
Added a test that when you first visit the site you get the Sign In button.

Reply:
Perfect thank you